src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -506,6 +506,14 @@ auto_flash(std::shared_ptr<xrt_core::device> & workingDevice, const std::string&
     }
     image_path = available_shells.front().second.get<std::string>("file");
   }
+  else if(boost::filesystem::exists(boost::filesystem::path(image))) {
+    update_shell(workingDevice->get_device_id(), image, std::string(""));
```

Comment:
I wonder whether you could just have
```suggestion
  else if (boost::filesystem::exists(image)) {
    update_shell(workingDevice->get_device_id(), image, {});
```

Reply:
This should work. Thanks